addons/test_mass_mailing/tests/test_mailing_sms.py

```diff
@@ -11,22 +11,22 @@
 from odoo.tools import mute_logger
 
 
-@tagged('mass_mailing', 'mass_mailing_sms')
+@tagged('mass_mailing', 'mass_mailing_sms', 'dato_test')
```

Comment:
Seems forgotten.

Fyi you can use this syntax to run single modules/classes/tests

`--test-tags /module:class.test` with every step optional
so 
`--test-tags :TestMassSMSInternals` here for example. Or `--test-tags .test_mass_sms_domain`